db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb

```diff
@@ -0,0 +1,13 @@
+class AddInOrdStatusCdToOrders < ActiveRecord::Migration
+  return if Order
```

Comment:
@wopr42, moved that check in all migrations... makes more sense being in the method.